mobile/src/main/res/values-de/strings.xml

```diff
@@ -118,4 +118,29 @@
     <string name="app_intro">Intro</string>
     <string name="intro_themes">Designs</string>
     <string name="intro_themes_description">Wählen Sie zwischen verschiedenen Designs</string>
+    <string name="favorites">Favoriten</string>
+    <string name="info_demo_mode_short">Läuft im Demomodus</string>
+    <string name="error_http_connection_failed">Ungültige Antwort von openHAB erhalten. Bitte überprüfen Sie openHAB (HTTP-Statuscode %d)</string>
+    <string name="error_certificate_not_trusted">SSL-Zertifikat nicht vertrauswürdig. Bitte importieren Sie das Root-Zertifikat oder deaktivieren Sie die Zertifikatsvalidierung</string>
+    <string name="error_certificate_expired">SSL-Zertifikat abgelaufen. Bitte erstellen Sie ein neues Zertifikat oder deaktivieren Sie die Zertifikatsvalidierung</string>
+    <string name="error_certificate_not_valid_yet">SSL-Zertifikat noch nicht gültig. Bitte erstellen Sie ein gültiges Zertifikat oder deaktivieren Sie die Zertifikatsvalidierung. Überprüfen Sie auch die Uhrzeit auf Ihrem Smartphone</string>
+    <string name="error_certificate_revoked">SSL-Zertifikat widerrufen. Bitte erstellen Sie ein neues Zertifikat oder deaktivieren Sie die Zertifikatsvalidierung</string>
+    <string name="error_http_code_401">Authentifizierung fehlgeschlagen. Bitte überprüfen Sie Nutzername und Passwort bzw. Client-Zertifikat (HTTP-Statuscode 401)</string>
+    <string name="error_http_code_403">Authentifizierung fehlgeschlagen. Bitte überprüfen Sie Nutzername und Passwort bzw. Client-Zertifikat (HTTP-Statuscode 403)</string>
+    <string name="error_http_code_407">Authentifizierung fehlgeschlagen. Bitte überprüfen Sie Nutzername und Passwort bzw. Client-Zertifikat (HTTP-Statuscode 407)</string>
+    <string name="error_http_code_414">openHAB URL zu lang (HTTP-Statuscode 414)</string>
+    <string name="error_http_code_426">HTTPS wird gefordet. Bitte ändern Sie die openHAB URL (HTTP-Statuscode 426)</string>
+    <string name="error_http_code_495">Ungültiges Client-Zertifikat (HTTP-Statuscode 495)</string>
+    <string name="error_http_code_496">Client-Zertifikat wird gefordert (HTTP-Statuscode 496)</string>
+    <string name="error_http_code_502">Ihr Reverse-Proxy bekam eine ungültige Antwort von openHAB (HTTP-Statuscode 502)</string>
+    <string name="error_http_code_504">Gateway Time-out. Ihr Reverse-Proxy läuft, aber bekam keine Antwort von openHAB (HTTP-Statuscode 504)</string>
+    <string name="error_http_code_507">openHAB hat nicht genug freien Speicher (HTTP-Statuscode 507)</string>
+    <string name="error_http_code_511">Netzwerkauthentifizierung wird gefordert (HTTP-Statuscode 511)</string>
+    <string name="ok">OK</string>
+    <string name="error_network_type_unsupported">Netzwerktyp (%s) wird nicht unterstützt</string>
```

Comment:
Was steht in dem eingesetzten String?

Reply:
Es wird geschaut, ob man wlan, lan oder mobiles internet hat. Wenn man beispielsweiße Tokenring hätte, würde es diesen fehler geben.

Reply:
Tokenring auf dem Android-Gerät? oO Klingt nach einem eher theoretischen Problem. ;-)

Ich meinte aber eher, durch was %s in der Praxis ersetzt wird.

Reply:
%s wird ersetzt mit der Ausgabe von https://developer.android.com/reference/android/net/NetworkInfo.html#getTypeName()

From here: https://developer.android.com/reference/android/net/ConnectivityManager.html
| | Type
-- | --
int | TYPE_BLUETOOTH           The Bluetooth data connection.
int | TYPE_DUMMY           Dummy data connection.
int | TYPE_ETHERNET           The Ethernet data connection.
int | TYPE_MOBILE           The Mobile data connection.
int | TYPE_MOBILE_DUN           A DUN-specific Mobile data connection.
int | TYPE_MOBILE_HIPRI                  This constant was deprecated       in API level 23.     Applications should instead use          requestNetwork(NetworkRequest, NetworkCallback) to request a network that          uses the TRANSPORT_CELLULAR transport.
int | TYPE_MOBILE_MMS                  This constant was deprecated       in API level 23.     Applications should instead use          requestNetwork(NetworkRequest, NetworkCallback) to request a network that          provides the NET_CAPABILITY_MMS capability.
int | TYPE_MOBILE_SUPL                  This constant was deprecated       in API level 23.     Applications should instead use          requestNetwork(NetworkRequest, NetworkCallback) to request a network that          provides the NET_CAPABILITY_SUPL capability.
int | TYPE_VPN           A virtual network using one or more native bearers.
int | TYPE_WIFI           The WIFI data connection.
int | TYPE_WIMAX           The WiMAX data connection.